pyproject.toml

```diff
@@ -10,7 +10,7 @@ addopts = """
   -rfEX -p pytester --strict-markers --verbosity=3
   """
 env = [
-    "CRATEDB_CONNECTION_STRING=crate://crate@localhost/?ssl=false",
+    "CRATEDB_CONNECTION_STRING=crate://crate@localhost/?ssl=false&schema=testdrive",
```

Comment:
Should `&schema=testdrive` be here or is it a leftover from testing?

Reply:
It is here to tell the Python driver to use that default schema when running the integration tests. So, it is intentionally there for testing, correct.